### ISSUE TITLE
Source problem

### ISSUE THREAD

**Issue by svencanotgay** [Proto-App/Proto-Android#36]
I found a problem where when I use an AC source, voltage and current go only from one output. So my twoway replifier doesnt work right.

**Comment by Proto-App** [comment]
@svencanotgay Could you describe your problem in more details? Or maybe post screenshot with your issue?

**Comment by svencanotgay** [comment]
Sorry my description wasnt good. So I made a replifier. I used Gretzkys two way replifier. Then I connected filtration (capacitator) and zeners diod as a stabilizator. Then I connected a LED. This was ok and function was fine but then I blown LED bc there was a big el voltage. So I took down voltage and repaird LED. After this the whole device stopped working. Everything was fine but the current stopped flowing from the source through the one way of replifier. Everything was connected and components were fine too. So I made a same device under the old one and it didnt work eaither. So I had to made another one but in new sketch and this one finally started wotk. Sorry for my en I am not a native speaker ;)
